efficientdet/inference.py

```diff
@@ -590,7 +668,8 @@ def __init__(self,
                num_classes: int = None,
                enable_ema: bool = True,
                data_format: Text = None,
-               label_id_mapping: Dict[int, Text] = None):
```

Comment:
How about hide the complexity inside det_post_process:

def det_post_process_batch_nms(...)

def det_post_process_single_nms(...)

def det_post_process_single(..., use_batch_nms=False):
  if use_batch_nms:
    return det_post_process_batch_nms(...)
  else:
    return det_post_process_single_nms(...)

In this case, all callers can be keep the same (only specify use_batch_nms)